po/de.po

```diff
@@ -113,7 +113,7 @@ msgstr "Adresse eingetragen."
 
 #: alias.c:394
 msgid "Error seeking in alias file"
-msgstr "Fehler beim Suchen in alias Datei"
+msgstr "Fehler beim Suchen in Addressbuchdatei"
```

Comment:
Strings from https://github.com/mineo/neomutt/blob/3d05bf38a3f9d99168ea1eac8736c2da9bc22c1e/po/de.po#L62 onwards translate alias-related things as Addressbuch, including https://github.com/mineo/neomutt/blob/3d05bf38a3f9d99168ea1eac8736c2da9bc22c1e/po/de.po#L107, which is the same as the string here except for a single word.